phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -105,60 +129,52 @@ public void testTooLowSCNWithMaxLookbackAge() throws Exception {
     public void testRecentlyDeletedRowsNotCompactedAway() throws Exception {
         try (Connection conn = DriverManager.getConnection(getUrl())) {
             String dataTableName = generateUniqueName();
-            String indexStem = generateUniqueName();
-            createTableAndIndexes(conn, dataTableName, indexStem);
-            String fullIndexName = indexStem + "1";
+            createTable(dataTableName);
+            injectEdge.incValue(WAIT_AFTER_TABLE_CREATION);
```

Comment:
nit: increamentValue

Reply:
The ManualEnvironmentEdge is copied from HBase code, where that's the method name. I couldn't use the HBase class because Phoenix's EEM is odd and requires a specific abstract subclass of Edge

Reply:
Given that code does not override the method, it makes sense to rename the method.

Reply:
Done.